client/src/partials/stock/stock.js

```diff
@@ -172,7 +179,8 @@ angular.module('bhima.controllers')
       }*/
 
       var path = config.modules.indexOf(defn) > -1 ? defn.link + '/' + $scope.depot.uuid
-        : (config.utilities.indexOf(defn) > -1 ) ? defn.link+ '/' + $scope.depot.uuid : defn.link;
+        : (config.utilities.indexOf(defn) > -1 ) ? defn.link+ '/' + $scope.depot.uuid 
+        : (config.reports.indexOf(defn) > -1 ) ? defn.link+ '/' + $scope.depot.uuid : defn.link;
```

Comment:
This is very hard to read.  What is happening here?  If it is possible to write out these routes explicitly, it would be easier for your fellow developers.